Source/Renderer/DrawCommand.js

```diff
@@ -32,6 +32,17 @@ define([
         this.boundingVolume = options.boundingVolume;
 
         /**
+         * The oriented bounding box of the geometry in world space. If this is defined, it is used instead of
+         * {@link DrawCommand#boundingVolume} for plane intersection testing.
+         *
+         * @type {OrientedBoundingBox}
+         * @default undefined
+         *
+         * @see DrawCommand#debugShowBoundingVolume
+         */
+        this.boundingOBB = options.boundingOBB;
```

Comment:
Yes, it is fine for debug code.  Just check to see if one of the unique properties is defined and include a comment saying why it is checking.